packages/sui-mono/src/config.js

```diff
@@ -35,6 +36,15 @@ module.exports = {
   },
   getPublishAccess: function () {
     return publishAccess
+  },
+  getProjectName: function () {
+    return projectPackage.name
+  },
+  isMonoPackage: function () {
+    const folders = cwds(path.join(basePath, packagesFolder), deepLevel)
+    const scopePath = folders.pop()
+
```

Comment:
Maybe deeplevel should be 0 by default and we should check this . Then we would have to add deeplevel =1 to current mono repos... but not a big deal and the rest would make sense IMO